✨Feature. launch screen 추가

pngxlols opened the pull request:
## ✨ What’s this PR?
### 📌 관련 이슈 (Related Issue)
- non, add launch screen

---

### 🧶 주요 변경 내용 (Summary)

- Resources 그룹에 LaunchScreen.storyboard 파일추가
- Saboteur 앱 타겟의 General → App Icons and Launch Screens → Launch Screen File에 LaunchScreen 지정

---

### 📸 스크린샷 (Optional)

https://github.com/user-attachments/assets/4df678f8-1bbf-4bc4-b295-536df814ed31
<img width="500" alt="런치스크린 정상 동작" src="https://github.com/user-attachments/assets/3069d559-62ca-4346-8fa4-72ed2ff07ad6" />


---

### 🧪 테스트 / 검증 내역

- [x] 앱 실행 시 Launch Screen 정상 표시 확인
- [x] 앱 정상 동작 확인

---

### 💬 기타 공유 사항

- 추후 디자인 변경 시 해당 스토리보드에서 바로 수정 가능
- 비에라가 만든 코드이기때문에 승인 받으면 assign 비에라가 할 수 있게 해뒀습니다~~

---

### 🙇🏻‍♀️ 리뷰 가이드 (선택)

- General 탭의 Launch Screen File 설정 위치

pngxlols committed:
✨Feature. launch screen 추가

bisor0627 commented on Saboteur/Info.plist:
```diff
@@ -2,13 +2,6 @@
-        <string>Maplestory OTF Light.otf</string>
-    </array>
-	<key>NSLocalNetworkUsageDescription</key>
-	<string>This application will use local networking to discover nearby devices.</string>
```
```
<key>NSLocalNetworkUsageDescription</key>
	<string>This application will use local networking to discover nearby devices.</string>
```


여기만 복구 부탁드려요!!

pngxlols replied:
앗 코드를 편집한게 아니라 general에 launch screen 추가했을 때 자동으로 변한거라 더 알아보고 넣을게요!

pngxlols replied:
파일이 계속 사라졌다 생겼다 해서 좀 오래걸렸네요
다시 넣어도 Saboteur 앱 정상적으로 돌아가서 반영해서 커밋했습니다!

pngxlols committed:
restore NSLocalNetworkUsageDescription

<key>NSLocalNetworkUsageDescription</key>
	<string>This application will use local networking to discover nearby devices.</string>
복구

bisor0627 reviewed (approved):
고생하셨어요 니케!
